src/views/mixed-content/top-countries-with-more-websites-with-mixed-content/index.js

```diff
@@ -0,0 +1,93 @@
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import React, {useState, useEffect} from 'react';
+import {api} from '../../../client';
+import {ChartContainer} from './elements';
+import {ResponsiveChoropleth} from '@nivo/geo';
+import * as features from './features.json';
+
+
+const TopCountriesWithMoreWebsitesWithMixedContent = () => {
+  const [loading, setLoading] = useState(true);
+  const [data, setData] = useState([]);
+
+  useEffect(() => {
+    api
+        .get('/api/mixed-content/top-countries' +
+        '-with-more-government-websites-with-mixed-content')
+        .then((response) => {
+          setData(response.data.result);
+          console.log('fetched');
+          setLoading(false);
+        })
+        .catch((err) => {
+          console.log(err);
+        });
+  }, []);
+
+  return (
+    <Card>
+      <CardContent>
+        <h2>Top countries with more government websites with mixed content</h2>
+        <p>Countries with more government websites that have mixed content</p>
+        <ChartContainer>
+          {!loading ? (
+            <ResponsiveChoropleth
+              data={data}
+              features={features.features}
+              margin={{top: 0, right: 0, bottom: 0, left: 0}}
+              colors="YlOrRd"
+              domain={[0, 10000]}
+              unknownColor="#666666"
```

Comment:
We might use the default Material UI theme to avoid hardcoding colors.

Check this out: https://material-ui.com//customization/theming/

You can import our theme from the theme folder at /src

This will allow us to have a standard.

Same for other charts